src/vs/workbench/contrib/debug/browser/debugQuickAccess.ts

```diff
@@ -63,7 +63,7 @@ export class StartDebugQuickAccessProvider extends PickerQuickAccessProvider<IPi
 						tooltip: localize('customizeLaunchConfig', "Configure Launch Configuration")
 					}],
 					trigger: () => {
-						config.launch.openConfigFile({ preserveFocus: false, useInitialConfigs: false });
+						config.launch.openConfigFile({ preserveFocus: false });
```

Comment:
I don't know why I added `useInitialConfigs: false` here, this should only happen for configs that already exist in a launch.json, so the file should already exist